### ISSUE TITLE
Symbol’s value as variable is void: hack--header-regex

### ISSUE THREAD

**Issue by jra3** [hhvm/hack-mode#15]
There seems to be an error with the current hack-mode when running (require 'hack-mode)

**Comment by Wilfred** [comment]
`hack--header-regex` is defined, and the test suite is passing. The variable did move in c6ef21eded65c699494642bc668d932e8ada2db8 though.

Could you describe how to replicate this?

**Comment by jra3** [comment]
seems to have cleared up... maybe there was a bad MELPA package yesterday...

I just reproed this, but blowing away my elpa packages and reinstalling fixed it. In any case... here *were* my steps. Cheers.

I've installed straight from MELPA

Started emacs with 

`/bin/emacs -Q`

`M-x emacs-version`
GNU Emacs 25.3.1 (x86_64-redhat-linux-gnu, X toolkit, Xaw scroll bars) of 2018-04-03

evaluated these lines
```
(require 'package)
(package-initialize)
(require 'hack-mode)
```
